ci/run-tests.sh

```diff
@@ -7,4 +7,5 @@ cargo test \
     --workspace \
     --exclude 'wasmtime-wasi-*' \
     --exclude wasi-crypto \
+    --exclude wasi-preview1-component-adapter \
```

Comment:
Instead of this could the manifest of the adapter say `test = false` for the `[lib]` section?